src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
+return(
     <div className="chat-entry local">
```

Comment:
I see you didn't do the optional enhancement, just in case you were wondering about how it worked you could basically use the ids of each message to decide if it should be on the right or left side of the screen. the receiver would always be an even id and the sender would always be odd. 

`const localRemote =
    props.id  % 2  === 0 ? 'chat-entry local' : 'chat-entry remote';
`

then  we could assign could put this class name in the `div` on line 8